pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -156,6 +151,14 @@ autoPatchelf() {
     cachedDependencies+=(
         $(find "$prefix" \! -type d \( -name '*.so' -o -name '*.so.*' \))
     )
+    # Also add all runtime dependencies
+    if [ -n "$runtimeDependencies" ]; then
```

Comment:
Where is `runtimeDependencies` defined anyways?

Reply:
It's defined by nix.